lib/date.js

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment';
+
+export function exhibitionPeriod(startAt, endAt) {
+  const startMoment = moment(startAt);
+  const endMoment = moment(endAt);
+  const thisMoment = moment();
+
+  let startFormat = 'MMMM Do';
+  if (startMoment.year() !== endMoment.year()) {
+    startFormat = startFormat.concat(', YYYY');
+  }
+
+  let endFormat = 'Do';
+  if (endMoment.year() !== thisMoment.year()) {
+    endFormat = endFormat.concat(', YYYY');
+  }
+  if (!(startMoment.year() === endMoment.year() && startMoment.month() === endMoment.month())) {
+    endFormat = 'MMMM '.concat(endFormat);
+  }
+
+  return `${startMoment.format(startFormat)} - ${endMoment.format(endFormat)}`;
+}
```

Comment:
Yah; I'm with it after thinking about it. I think historically this has been a problem because of a lack of consistency site-wide when designs have been specified but there's now a move to actually consolidate. See the image of the spec posted below. It'll be nice to not have to worry about it for sure.